nextroute/check/options.go

```diff
@@ -0,0 +1,9 @@
+package check
+
+import "time"
+
+// Options are the options for a check.
+type Options struct {
+	Duration  time.Duration `json:"duration" usage:"maximum duration of the check" default:"30s"`
+	Verbosity string        `json:"verbosity" usage:"verbosity of the check, current available options are 'off' , 'low', 'medium', 'high', 'very_high'" default:"off"`
```

Comment:
```suggestion
	Verbosity string        `json:"verbosity" usage:"{off, low, high, very_high} verbosity of the check" default:"off"`
```

This is how we display set member with solutions all/last and verbosity in `mip` as well.